draft-ietf-quic-http.md

```diff
@@ -122,13 +194,65 @@ Field definitions are given in Augmented Backus-Naur Form (ABNF), as defined in
 This document uses the variable-length integer encoding from
 {{QUIC-TRANSPORT}}.
 
-Protocol elements called "frames" exist in both this document and
-{{QUIC-TRANSPORT}}. Where frames from {{QUIC-TRANSPORT}} are referenced, the
-frame name will be prefaced with "QUIC."  For example, "QUIC CONNECTION_CLOSE
-frames."  References without this preface refer to frames defined in {{frames}}.
+The following terms are used:
+
+abort:
+: An abrupt termination of a connection or stream, possibly due to an error
+  condition.
+
+client:
+: The endpoint that initiates an HTTP/3 connection.  Clients send HTTP requests
+  and receive HTTP responses.
+
+connection:
+: A transport-layer connection between two endpoints, using QUIC as the
+  transport protocol.
+
+connection error:
+: An error that affects the entire HTTP/3 connection.
+
+endpoint:
+: Either the client or server of the connection.
+
+frame:
+: The smallest unit of communication within an HTTP/3 connection, consisting of
```

Comment:
I know this comes from HTTP/2 but one could argue that frames are not the smallest unit in HTTP/3.